nens_auth_client/requests_session.py

```diff
@@ -0,0 +1,103 @@
+from .models import RemoteUser
+from .oauth import get_oauth_client
+from requests import Session
+from typing import Optional, List, Union
+from django.conf import settings
+
+
+def refresh_token(remote_user: RemoteUser):
+    client = get_oauth_client()
+    tokens = client.fetch_access_token(
+        refresh_token=remote_user.refresh_token, grant_type="refresh_token"
```

Comment:
I think the refresh_token can also timeout. However in that case the user is totally logged out.

Reply:
Yes you get an `OAuthError` with detail "invalid_grant" in that case. Just tested it (I set the refresh token expiry to 1 hour yesterday)
I'll document that.